test/cookbooks/verify-octo/recipes/tentacle.rb

```diff
@@ -18,10 +18,39 @@
 # limitations under the License.
 #
 
+tentacle = node['verify-octo']['tentacle']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.